CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+# Fabric8-Launcher Backend Contributing Guide
+
+If you're looking for a development guide for the Fabric8-Launcher Backend, first read [the readme](./README.md).
+
+## First Steps
+
+### Prerequisites
+
+* Java 8+
+* Maven 3.5.3+
+* Git
+
+### GitHub Workflow
+
+To contribute to the Fabric8-Launcher Backend:
+
+1. Fork this repository to your own Git.
+2. Clone your fork.
+3. Commit/Push your work on topic branches
+4. Make pull requests.
+
+Here are 3 simple rules we follow:
+- Use a fork of our repository with a topic branch to contribute.
+- Redact a beautiful commit/pull-request message following our [guide](./COMMIT_MSG.md).
+- Try to keep a clean commit log (no unnecessary commits, no merge commits).
+
+In order to make you feel safe regarding your contribution, we are here to help you:
+ - Unit/Integration tests and checkstyle are run for every modification you make.
+ - There is a test keeper plugin so you don't forget to test your changes if necessary.
```

Comment:
Is this line necessary? It seems very specific.
(And I'm already getting annoyed by this stupid plugin, so let's see how long it survives :-p )

Reply:
@quintesse lol I was so sure you would get annoyed by the plugin that I want to keep this line just for you :)